modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Answer.es.js

```diff
@@ -84,6 +84,7 @@ export default withRouter(
 					className={classnames('questions-answer c-p-3', {
 						'questions-answer-success': showAsAnswer,
 					})}
+					data-testid="mark-as-answer-style"
```

Comment:
it would be great if you could reuse the existing class

Reply:
Do you mean `questions-answer-success`? The CSS class? Or only reuse the name?